UnitTest1.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using Medicine;

```

Comment:
Имя пространства и мен неверное. У тестируемой библиотеки пространство имен другое
```suggestion
using задание11;

```